environment.yml

```diff
@@ -6,7 +6,6 @@ channels:
   - bioconda
   - defaults
 dependencies:
-  - conda-forge::python=3.7.3
```

Comment:
shouldn't here the python 2 version be specified instead?

Reply:
Python 2.7 is already requested by manta
So not to clutter the environment.yml file, I didn't specified it
Same with r-base etc...

Reply:
ok, good then!